src/components/Image/Image.tsx

```diff
@@ -1,17 +1,26 @@
+import { useEffect, useState } from "react";
 import chuckImg from "../../assets/img/chuck.png";
+import unkownImg from '../../assets/img/unknown.png'
```

Comment:
typo 😸 install vscode extension for typo checking https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker